src/common/exif.cc

```diff
@@ -1110,6 +1110,19 @@ int dt_exif_write_blob(uint8_t *blob, uint32_t size, const char *path, const int
       dt_remove_exif_keys(imgExifData, keys, n_keys);
     }
 
+    // remove the profile tables
```

Comment:
@mazhe
Uh, now that i actually read this, shouldn't it go into `dt_exif_read_blob()`, into `if(!dng_mode) {}` ?